### PR TITLE
Add user prompt to scripts

### PR DESCRIPTION
**Pull request by ropow**

### CODE REVIEW

File: SetupCrossTenantRelationshipForResourceTenant.ps1 [hunk 1]
```diff
@@ -108,5 +109,20 @@ function Run-ExchangeSetupForResourceTenant([string]$targetTenant, [string]$targ
             -MailboxMovePublishedScopes $sourceMailboxMovePublishedScopes
     }
 }
+function UserPrompt() {
+    Write-Host `n
+    Write-Host "Welcome to the Cross-tenant mailbox migration preview! Before running this script, please be sure to review the details provided on docs.microsoft.com at the following URL: `nhttps://docs.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-mailbox-migration"
+    Write-Host "It is also recommended before running this script to review the script in a script editor or Notepad prior to running."`n
+    Write-Host "For general feedback and / or questions, please contact crosstenantmigrationpreview@service.microsoft.com.`nThis is not a support alias and should not be used if you are currently experiencing an issue and need immediate assistance."`n
+    $title = "If you are ready to begin configuring your tenants, select 'Y'. If you need to review any additional details and proceed at a later time, select 'N'."
```
Comment (Andy-0-Ryan) [inline SetupCrossTenantRelationshipForResourceTenant.ps1:117]: I think I'd refactor this, try it out in PS UI too.. This looks better

$title = "Confirm : Configure Cross-tenant mailbox migration preview"
    $message = "If you are ready to begin configuring your tenants, select 'Y'.`nIf you need to review any additional details and proceed at a later time, select 'N'.`n`nDo you wish to proceed?"

File: SetupCrossTenantRelationshipForTargetTenant.ps1 [hunk 2]
```diff
@@ -595,7 +595,23 @@ if (-not $enumExists) {
 "@
 }
 
-Main
+function UserPrompt() {
+    Write-Host `n
+    Write-Host "Welcome to the Cross-tenant mailbox migration preview! Before running this script, please be sure to review the details provided on docs.microsoft.com at the following URL: `nhttps://docs.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-mailbox-migration"
+    Write-Host "It is also recommended before running this script to review the script in a script editor or Notepad prior to running."`n
+    Write-Host "For general feedback and / or questions, please contact crosstenantmigrationpreview@service.microsoft.com.`nThis is not a support alias and should not be used if you are currently experiencing an issue and need immediate assistance."`n
+    $title = "If you are ready to begin configuring your tenants, select 'Y'. If you need to review any additional details and proceed at a later time, select 'N'."
```
Comment (Andy-0-Ryan) [inline SetupCrossTenantRelationshipForTargetTenant.ps1:603]: Same as other::
$title = "Confirm : Configure Cross-tenant mailbox migration preview"
    $message = "If you are ready to begin configuring your tenants, select 'Y'.`nIf you need to review any additional details and proceed at a later time, select 'N'.`n`nDo you wish to proceed?"

**Review by Andy-0-Ryan — CHANGES_REQUESTED** [review]
Nit

**Review by psykokwak4 — APPROVED** [review]

File: SetupCrossTenantRelationshipForTargetTenant.ps1 (hunk 2 above)
Comment (ropow) [reply SetupCrossTenantRelationshipForTargetTenant.ps1:603]: I don't really see a need for a line break here? In my powershell window it looks fine. Can you give me an example of why this is needed.

Comment (ropow) [reply SetupCrossTenantRelationshipForTargetTenant.ps1]: Makes sense, this does look better.

**Pull request merged**